samples/client/petstore/spring-cloud-async/src/main/java/org/openapitools/model/Category.java

```diff
@@ -15,9 +15,11 @@
 @ApiModel(description = "A category for a pet")
 
 public class Category   {
+
```

Comment:
@MarkusBansky Looks like your code formatter made a LOT of whitespace changes... Perhaps it would be easier to review this PR if there weren't 500+ files with simple whitespace changes?

Reply:
@InfoSec812 these files where generated from a terminal, with a command from the pull request description, why is it inserting a whitespace then?

Reply:
It could be that your changes to the template is causing the whitespace to be added?

Reply:
I played around with this. It is because of the mustache template. I tried this:
```diff --git a/modules/openapi-generator/src/main/resources/JavaSpring/pojo.mustache b/modules/openapi-generator/src/main/resources/JavaSpring/pojo.mustache
index e26e1d8d9b..5ccb28faf8 100644
--- a/modules/openapi-generator/src/main/resources/JavaSpring/pojo.mustache
+++ b/modules/openapi-generator/src/main/resources/JavaSpring/pojo.mustache
@@ -19,8 +19,10 @@ public class {{classname}} {{#parent}}extends {{{parent}}}{{/parent}}{{^parent}}
 {{>enumClass}}
     {{/mostInnerItems}}
     {{/isContainer}}
-    {{/isEnum}}{{#additionalAnnotations}}
-  {{{.}}}{{/additionalAnnotations}}
+    {{/isEnum}}
+{{#additionalAnnotations}}
+  {{{.}}}
+{{/additionalAnnotations}}
   {{#jackson}}
   @JsonProperty("{{baseName}}"){{#withXml}}
   @JacksonXmlProperty({{#isXmlAttribute}}isAttribute = true, {{/isXmlAttribute}}{{#xmlNamespace}}namespace="{{xmlNamespace}}", {{/xmlNamespace}}localName = "{{#xmlName}}{{xmlName}}{{/xmlName}}{{^xmlName}}{{baseName}}{{/xmlName}}"){{/withXml}}

```
And that clean it up.

Reply:
Is there any progress on reverting newline/whitespaces from this PR? As this feature is somewhat I've been looking for, I'm very much intrested in it. I was looking into customizing spring generator by myself and it seems almost everything I need is ready. 
Maybe one could help with sanitizing those 500+ newline changes or reimplementing the PR?